src/components/SignUpForm.tsx

```diff
@@ -1,69 +1,64 @@
-import React, { useState } from 'react';
+import React, { useCallback, useState } from 'react';
 import Form from 'react-bootstrap/Form';
 import Button from 'react-bootstrap/Button';
 
-function SignUpFormHeader() {
+const SignUpFormHeader = () => {
   return <h3>Luo uusi käyttäjä</h3>;
-}
+};
 
-function SignUpForm() {
+const SignUpForm = () => {
   const [values, setValues] = useState({
     firstName: '',
     lastName: '',
-    userId: '',
+    username: '',
```

Comment:
I started to wonder in shower, if we really even need a `username`. Where would that be used? We probably want to use email as the login identifier, since that is one less thing for the users to remember.